ibis/expr/types/generic.py

```diff
@@ -1175,19 +1175,19 @@ def group_concat(
         │           36.7 │          19.3 │
         └────────────────┴───────────────┘
         >>> t.bill_length_mm.group_concat()
-        ┌───────────────────────┐
-        │ '39.1,39.5,40.3,36.7' │
-        └───────────────────────┘
+        ┌─────────────────────┐
+        │ 39.1,39.5,40.3,36.7 │
+        └─────────────────────┘
 
         >>> t.bill_length_mm.group_concat(sep=": ")
-        ┌──────────────────────────┐
-        │ '39.1: 39.5: 40.3: 36.7' │
-        └──────────────────────────┘
+        ┌────────────────────────┐
+        │ 39.1: 39.5: 40.3: 36.7 │
+        └────────────────────────┘
 
         >>> t.bill_length_mm.group_concat(sep=": ", where=t.bill_depth_mm > 18)
-        ┌──────────────┐
-        │ '39.1: 36.7' │
-        └──────────────┘
+        ┌────────────┐
+        │ 39.1: 36.7 │
```

Comment:
Do the string `"90"` and the number `90` repr the same now?

Reply:
Yes, but that's the same as they do for columns as well.

Reply:
I suppose we could quote strings for scalars only?

Reply:
I guess the main difference is that with the column you get the type information.

-0 on losing the quotes, but not a blocker.

Reply:
Eh, let's just leave it and see how it feels to use it.